archstrike-testing/smtp-fuzz/PKGBUILD

```diff
@@ -4,18 +4,19 @@ buildarch=1
 
 pkgname=smtp-fuzz
 pkgver=1.0
-pkgrel=3
-epoch=1
```

Comment:
epoch needs to stay, or it will cause versioning issues.

Reply:
done